controllers/networking-calico/charts/internal/calico/values.yaml

```diff
@@ -4,7 +4,9 @@ config:
   backend: bird
   ipam:
     type: "host-local"
-#    subnet: "usePodCidr"
+#   subnet: "usePodCidr"
+  typha: {}
+#   enabled: true
```

Comment:
Can we uncomment this as this should be the default? I know, it's overwritten when actually rendering the chart, though, just to be safe.

Reply:
done